Automattic/wc-calypso-bridge
# Free trial: Replace Marketing page

### Changes proposed in this Pull Request:



Blocked:

1. Waiting for the main image: lD0gTB4IqGsmSmsnjZNaP6-fi-1096:24909
2. Waiting for `Create digital gift cards` link
3. ~I'm going to see if I can optimize SVG files to reduce the bundle size.~

Closes #953 

This PR replaces Marketing page from the core with a newly designed Marketing page for free trial plan.




### How to test the changes in this Pull Request:

1. Checkout the branch and build the assets.
2. Go to admin page
3. Confirm the Marketing menu doesn't have any submenus
4. Click on the Marketing menu.
5. Confirm the new design.



### Other information:




### FOR PR REVIEWER ONLY:
